build.info

```diff
@@ -1,4 +1,8 @@
+IF[{- $config{options} =~ /no-libssl/ -}]
```

Comment:
And now I have to wonder how I failed to find the existing usage of that construct in other build.info files.  Thanks.